pilot/pkg/security/authz/builder/builder.go

```diff
@@ -115,18 +114,18 @@ func build(policies []model.AuthorizationPolicy, tdBundle trustdomain.Bundle, ac
 		for i, rule := range policy.Spec.Rules {
 			name := fmt.Sprintf("ns[%s]-policy[%s]-rule[%d]", policy.Namespace, policy.Name, i)
 			if rule == nil {
-				authzLog.Errorf("skipped nil rule %s", name)
+				authzLog.Errorf(isOnInboundPassthrough, "skipped nil rule %s", name)
```

Comment:
this and below should not be changed, it should still print the valid error logs instead of skipping all of them for pass through filter chain.